mods/ts/rules/defaults.yaml

```diff
@@ -566,6 +569,9 @@
 	Carryable:
 		RequiresCondition: !inside-tunnel
 	RevealOnFire:
+	RejectsOrders@tunnel:
+		Reject: Stop
```

Comment:
IMO we would ideally not reject, but defer these orders so that they stop/deploy/etc as soon as they exit the tunnel.  This should be possible, in principle, by making the `Move` activity uncancellable if `!mobile.CanInteractWithGroundLayer`.  Knowing the current state of the `Move` code, I would be surprised if it was really that simple (ping: @obrakmann).

`RejectsOrders` is a good enough stopgap for now, and the improvements to that trait are certainly worthwhile for other use cases.